### DISCUSSION TITLE
New network (Polygon Mumbai Testnet)

### DISCUSSION THREAD

**Discussion by dersanli** [rainbow-me/rainbowkit#421] [Feedback]
Hi guys is there a way to add Polygon Mumbai Testnet (80001) to the Rainbow Wallet app?

**Comment by nickbytes** [comment]
Going to close this issue since it's not specific to RainbowKit. Passed along to the team that this would be helpful though! Appreciate you testing out RainbowKit and being a Rainbow user!
